resources/authgear/templates/ja/translation.json

```diff
@@ -0,0 +1,1041 @@
+  "territory-MU": "モーリシャス",
+  "territory-MV": "モルディブ",
+  "territory-MW": "マラウイ",
+  "territory-MX": "メキシコ",
+  "territory-MY": "マレーシア",
+  "territory-MZ": "モザンビーク",
+  "territory-NA": "ナミビア",
+  "territory-NC": "ニューカレドニア",
+  "territory-NE": "ニジェール",
+  "territory-NF": "ノーフォーク島",
+  "territory-NG": "ナイジェリア",
+  "territory-NI": "ニカラグア",
+  "territory-NL": "オランダ",
+  "territory-NO": "ノルウェー",
+  "territory-NP": "ネパール",
+  "territory-NR": "ナウル",
+  "territory-NU": "ニウエ",
+  "territory-NZ": "ニュージーランド",
+  "territory-OM": "オマーン",
+  "territory-PA": "パナマ",
+  "territory-PE": "ペルー",
+  "territory-PF": "フランス領ポリネシア",
+  "territory-PG": "パプアニューギニア",
+  "territory-PH": "フィリピン",
+  "territory-PK": "パキスタン",
+  "territory-PL": "ポーランド",
+  "territory-PM": "サンピエール・ミクロン",
+  "territory-PN": "ピトケアン諸島",
+  "territory-PR": "プエルトリコ",
+  "territory-PS": "パレスチナ領土",
+  "territory-PT": "ポルトガル",
+  "territory-PW": "パラオ",
+  "territory-PY": "パラグアイ",
+  "territory-QA": "カタール",
+  "territory-QO": "外洋州",
+  "territory-RE": "レユニオン",
+  "territory-RO": "ルーマニア",
+  "territory-RS": "セルビア",
+  "territory-RU": "ロシア",
+  "territory-RW": "ルワンダ",
+  "territory-SA": "サウジアラビア",
+  "territory-SB": "ソロモン諸島",
+  "territory-SC": "セーシェル",
+  "territory-SD": "スーダン",
+  "territory-SE": "スウェーデン",
+  "territory-SG": "シンガポール",
+  "territory-SH": "セントヘレナ",
+  "territory-SI": "スロベニア",
+  "territory-SJ": "スヴァールバル諸島およびヤンマイエン島",
+  "territory-SK": "スロバキア",
+  "territory-SL": "シエラレオネ",
+  "territory-SM": "サンマリノ",
+  "territory-SN": "セネガル",
+  "territory-SO": "ソマリア",
+  "territory-SR": "スリナム",
+  "territory-SS": "南スーダン",
+  "territory-ST": "サントメ・プリンシペ",
+  "territory-SV": "エルサルバドル",
+  "territory-SX": "シントマーテン",
+  "territory-SY": "シリア",
+  "territory-SZ": "エスワティニ",
+  "territory-TA": "トリスタンダクーニャ",
+  "territory-TC": "タークス・カイコス諸島",
+  "territory-TD": "チャド",
+  "territory-TF": "フランス領南方領土",
+  "territory-TG": "トーゴ",
+  "territory-TH": "タイ",
+  "territory-TJ": "タジキスタン",
+  "territory-TK": "トケラウ",
+  "territory-TL": "東ティモール",
+  "territory-TM": "トルクメニスタン",
+  "territory-TN": "チュニジア",
+  "territory-TO": "トンガ",
+  "territory-TR": "トルコ",
+  "territory-TT": "トリニダード・トバゴ",
+  "territory-TV": "ツバル",
+  "territory-TW": "台湾",
+  "territory-TZ": "タンザニア",
+  "territory-UA": "ウクライナ",
+  "territory-UG": "ウガンダ",
+  "territory-UM": "アメリカ領有小離島",
+  "territory-UN": "国連",
+  "territory-US": "アメリカ合衆国",
+  "territory-UY": "ウルグアイ",
+  "territory-UZ": "ウズベキスタン",
+  "territory-VA": "バチカン",
+  "territory-VC": "セントビンセントおよびグレナディーン諸島",
+  "territory-VE": "ベネズエラ",
+  "territory-VG": "英領バージン諸島",
+  "territory-VI": "U.S. バージン諸島",
+  "territory-VN": "ベトナム",
+  "territory-VU": "バヌアツ",
+  "territory-WF": "ワリス・フツナ",
+  "territory-WS": "サモア",
+  "territory-XA": "疑似アクセント",
+  "territory-XB": "撮影のバイド",
+  "territory-XK": "コソボ",
+  "territory-YE": "イエメン",
+  "territory-YT": "マヨット",
+  "territory-ZA": "南アフリカ",
+  "territory-ZM": "ザンビア",
+  "territory-ZW": "ジンバブエ",
+  "territory-ZZ": "不明な地域",
+  "language-af": "アフリカーンス語",
```

Comment:
language-* should be kept untranslated. The names are intentionally kept untranslated so that people who know that language can identify it.